python-threatexchange/threatexchange/content_type/content_base.py

```diff
@@ -7,7 +7,7 @@
 This records all the valid signal types for a piece of content.
 """
 
-from enum import Enum, auto
+from enum import Enum
```

Comment:
remove

Reply:
Do you mean to keep the `import auto` line?

Reply:
Whoops, mistake on my part, thought this was an unused import rather than a change! Please ignore!